src/components/BooksContainer.jsx

```diff
@@ -16,7 +16,10 @@ const BooksContainer = () => {
       <section className="books-container">
         {booksList.map((book) => <Book key={book.item_id} bookObject={book} />)}
       </section>
-      <section className="form container"><Form /></section>
+      <section className="form-container">
+        <h3 className="form-header">ADD NEW BOOK</h3>
```

Comment:
- [ ] This header needs to be in one line, currently the width is not big enough, kindly update it

Yours: 
![image](https://github.com/adeola003/bookstore/assets/62705947/ecdec097-7635-4a11-a51f-ca01f75ec25b)
Design:
![image](https://github.com/adeola003/bookstore/assets/62705947/f17446d9-547c-41e9-9d38-8a10ecfb8aa5)

Reply:
Its one line by my side @GzimAsani

Reply:
- Hey @adeola003, ignore this comment I made the check on Firefox and for some reason it was showing like that, just double checked on chrome and it looks fine. I apologize for the confusion. Just make sure to fix the only comment about the button to make the text more on the  center